packages/quantic/package.json

```diff
@@ -56,7 +56,7 @@
     "@octokit/graphql-schema": "10.74.2",
     "@salesforce/eslint-config-lwc": "3.3.3",
     "@salesforce/eslint-plugin-lightning": "1.0.0",
-    "@salesforce/sfdx-lwc-jest": "1.4.1",
+    "@salesforce/sfdx-lwc-jest": "3.0.0",
```

Comment:
@coveo/salesforce-integration, UT were 💥 with my changes. I think it was because sfdx-lwc-jest v1 was not compatible with jest 29.
I naively bumped and seems to work.